ExpectedSlotConsoleMessages.json

```diff
@@ -1,7 +1,69 @@
-{
-    "SlotMessages": [
-      "gameLoadComplete",
-      "message type: \"balanceUpdate\"",
-      "message type: \"stakeUpdate\""
+  {
```

Comment:
The file is neat, but I am not sure how setting an array would be helpful in the actual test. For instance to test gbp, the user id will be different from the one being used for eur. How would we iterate through the array?
I think many of them will need to be separate items instead of arrays.

Reply:
I see what you mean, so it would be something like
`message === testmessages.supportedcurrencies.gbpmessage`
I think that would work nicely. So one test per currency will use one message